commit ebc14dc2e4ffa4a845df7aca955e98a740ccf69f (refs/changes/59/959/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2016-11-28T15:51:48-08:00 (2 years, 10 months ago)
    
    joyent/sdcadm#28 --force-data-path option is not applicable for `sdcadm create portolan` (update changelog and bumpver)

diff --git a/CHANGES.md b/CHANGES.md
index 08db4f1..9ffa95a 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -10,6 +10,10 @@
 
 # sdcadm Changelog
 
+## 1.14.1
+
+- joyent/sdcadm#28 Allow 'sdcadm create portolan --skip-ha-check' to work.
+
 ## 1.14.0
 
 - TOOLS-1610 Drop the "-$buildstamp" suffix on the sdcadm image version field
diff --git a/package.json b/package.json
index 915828a..9019689 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "sdcadm",
   "description": "Administer a SmartDataCenter (SDC) standup",
-  "version": "1.14.0",
+  "version": "1.14.1",
   "author": "Joyent (joyent.com)",
   "private": true,
   "dependencies": {
